### PR TITLE
#7540

### PR DESCRIPTION
**Pull request by HeorhiiBoliuk**
Add missing styles on overview grid block, change styles for overview block, add missing fields in template of overview grid block, on front page. Modify a node template. Add a title to overview and styles for pager and block.

### CODE REVIEW

File: web/themes/custom/my_awesome_theme/components/02-molecules/block-category-media-title/_block-category-media-title.scss [hunk 1]
```diff
@@ -71,15 +71,17 @@
...
+          a{
+            padding: 3px 7px;
+            font-family: 'Roboto',sans-serif;
+            font-size: 27px;
+            font-weight: 500;
+            line-height: 34px;
+            text-transform: uppercase;
+            pointer-events: auto;
+            color: $white;
+            text-decoration: none;
```
Comment (ivandudardb) [inline web/themes/custom/my_awesome_theme/components/02-molecules/block-category-media-title/_block-category-media-title.scss:83]: You can use @mixin here

File: web/themes/custom/my_awesome_theme/components/02-molecules/block-category-media-title/_block-category-media-title.scss [hunk 2]
```diff
@@ -147,15 +149,17 @@
...
+          a{
+            padding: 3px 7px;
+            font-family: 'Roboto',sans-serif;
+            font-size: 24px;
+            line-height: 30px;
+            font-weight: 500;
+            text-transform: uppercase;
+            pointer-events: auto;
+            color: $white;
+            text-decoration: none;
```
Comment (ivandudardb) [inline web/themes/custom/my_awesome_theme/components/02-molecules/block-category-media-title/_block-category-media-title.scss:161]: You can use @mixin here

File: web/themes/custom/my_awesome_theme/components/02-molecules/block-category-media-title/_block-category-media-title.scss [hunk 3]
```diff
@@ -225,15 +229,17 @@
...
+          a{
+            padding: 3px 7px;
+            font-family: 'Roboto',sans-serif;
+            font-size: 16px;
+            line-height: 21px;
+            font-weight: 500;
+            text-transform: uppercase;
+            pointer-events: auto;
+            color: $white;
+            text-decoration: none;
```
Comment (ivandudardb) [inline web/themes/custom/my_awesome_theme/components/02-molecules/block-category-media-title/_block-category-media-title.scss:241]: You can use @mixin here

**Review by ivandudardb — CHANGES_REQUESTED** [review]

**Review by ivandudardb — APPROVED** [review]
LGTM

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**